base/weakkeydict.jl

```diff
@@ -92,6 +92,7 @@ function getkey(wkh::WeakKeyDict{K}, kk, default) where K
     end
 end
 
+Base.map!(f,iter::Base.ValueIterator{WeakKeyDict{K,V}}) where {K, V} = Base.map!(f,values(iter.dict.ht))
```

Comment:
Just `iter::ValueIterator{<:WeakKeyDict}`, and no need for `Base.` qualifiers since this is in `Base` already.